scripts/build.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+require(__dirname + '/../' + 'constants');
+
+logger.info(collection);
+/*
+ * choices of collection to be filled
+ */
+var possibleListeners = [
+    'basicPokemonDetail'
+];
+
+// when the choices of collection to be filled doesn't match, then exit the process
+if (possibleListeners.indexOf(collection) < 0) {
+    process.exit();
+}
+
+// listener for pokemons
+const listener = require(__base + 'app/controllers/filler/' + collection);
+listener.fill(function () {
```

Comment:
change the names since it's not listener, change the name accordingly for build script.